internal/testrunner/runners/system/runner.go

```diff
@@ -487,6 +487,7 @@ func createPackageDatastream(
 	// Add package-level vars
 	pkgVars := kibana.Vars{}
 	input := pkg.PolicyTemplates[0].FindInputByType(streamInput)
+	input.Vars = append(input.Vars, pkg.Vars...)
```

Comment:
@ycombinator could you please verify this fix? I'm not quite sure about implications.

Reply:
My understanding of this change is that all the package-level `vars`, defined in the package's `manifest.yml` file, are being copied into to each input's `vars` list. 

From a system test definition perspective, this means that the top-level `vars` in the system test definition file could be used to override not just input vars but now also the package `vars`. So I think this change makes sense.

The only thing I would do is move this line inside the `if input != nil` check.

Reply:
Done! Thanks @ycombinator for the review!!